server_setup_scripts/setup_server.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
why is there both [scripts/setup_server.sh] and [server_setup_scripts/setup_server.sh]?

Reply:
I've renamed scripts to server_setup_scripts. Scripts shouldn't exist anymore

Reply:
Oh i guess that didn't go through, will fix